src/Http/Productions/ProductionsWriteController.php

```diff
@@ -22,13 +23,19 @@ class ProductionsWriteController
      * @var CreateProductionValidator
      */
     private $createProductionValidator;
+    /**
```

Comment:
```suggestion

    /**
```

(Missing a newline above the docblock here)